.github/PULL_REQUEST_TEMPLATE/rfc.md

```diff
@@ -30,22 +27,23 @@ Design PR: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra
 
 Zebra Issue: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/issues/0000)
 
-### Document
+### Rendered
 
 <!--
-Fill this in, replacing:
-
+Replace:
 ZcashFoundation/zebra with username/repo (if not making the PR from the Zebra repo)
 my-branch-name with the PR branch
 xxxx-my-feature with the filename of the RFC
-
 -->
 [Rendered](https://github.com/ZcashFoundation/zebra/blob/my-branch-name/book/src/dev/rfcs/drafts/xxxx-my-feature.md).
 
 ## Zebra Team Approval
 
-Everyone on the Zebra team should review design RFCs:
+Most of the Zebra team should review design RFCs:
 
+- [ ] @conradoplg
 - [ ] @dconnolly
 - [ ] @oxarbitrage
+- [ ] @jvff
+- [ ] @mpguerra
 - [ ] @teor2345
```

Comment:
```suggestion
- [ ] @teor2345
- [ ] @upbqdn 
```
I added Marek :)

Reply:
I left Marek off because he's not full-time right now, and I'm not sure how much he will be working on Zebra.
(Every time we open a RFC PR it will ping him.)

@mpguerra do you want Marek notified on every Zebra RFC design PR?

Reply:
(I've put the PR in draft until we resolve this.)

Reply:
Marek should be full time from Sprint 12 onwards, I think eventually I would like him to work on Zebra although maybe not starting on Sprint 12

Reply:
Ok, let's wait to add Marek until he's actually working on Zebra.

Reply:
(I force-pushed to remove this change.)